src/BizHawk.Emulation.Cores/Consoles/Nintendo/BSNES/SubBsnesCore.cs

```diff
@@ -0,0 +1,161 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using BizHawk.Emulation.Common;
+
+namespace BizHawk.Emulation.Cores.Nintendo.BSNES
+{
+	[PortedCore(CoreNames.SubBsnes115, "")]
+	[ServiceNotApplicable(new[] { typeof(IDriveLight) })]
+	public class SubBsnesCore
+		: IEmulator, IDebuggable, IVideoProvider, ISaveRam, IStatable, IInputPollable, IRegionable,
```

Comment:
You're registering most of these services from the BsnesCore instance anyways, don't re-implement them here (except ICycleTiming, which you don't want "implemented" in the non-subframe core)